docs/man_pages/publishing/appmanager-upload-android.md

```diff
@@ -13,9 +13,14 @@ WARNING: This command is not applicable to mobile website projects. To view the
 ### Options
 * `--certificate` - Sets the certificate that you want to use for code signing your Android app. You can set a certificate by index or name. <% if(isHtml) { %>To list available certificates, run `$ appbuilder certificate`<% } %> 
 * `--download` - If set, downloads the application package to the root of the project.
+* `--publish` - If set, the application package will be automatically published and visible for the members of its distribution groups. To add distribution groups, use `--groups` option.
+* `--send-email` - If set, an email with the link to the app will be send to all eligible users.
```

Comment:
what will happen in case I specify --send-email, but I do not specify --publish? Will we send email to users?Will the command fail?
In case --send-email and --send-push are not applicable when --publish is used, consider adding a check and warning message (or even fail) inside your code, to inform the user for the correct way to use the options.